### PR TITLE
Sup 10006 use p tag instead of br for captions

### PR DESCRIPTION
**Pull request by omridevk**

### CODE REVIEW

**Comment by ErezBouskila** [comment]
@OrenMe

File: modules/TimedText/resources/mw.TextSource.js [hunk 1]
```diff
@@ -438,7 +438,9 @@
 					}
 					if( caption ){
 						// concatenate text lines to html text
-						caption['content'] = s.slice(2).join("<br>");
+						caption['content'] = s.slice(2).reduce(function(ac, current) {
```
Comment (OrenMe) [inline modules/TimedText/resources/mw.TextSource.js]: IE8 doesn't support reduce method

Comment (omridevk) [inline modules/TimedText/resources/mw.TextSource.js]: can't we add the Polyfill for reduce?

Comment (OrenMe) [inline modules/TimedText/resources/mw.TextSource.js]: @omridevk is this an array slice and concat? I believe you can achieve the same result with array concat woth seperator

**Comment by OrenMe** [comment]
Solved in https://github.com/kaltura/mwEmbed/pull/3280

**Pull request closed without merging**